cmd/kubeapps-apis/server/plugins_test.go

```diff
@@ -240,3 +243,120 @@ func createTestFS(t *testing.T, filenames []string) fstest.MapFS {
 	}
 	return fs
 }
+
+func TestExtractToken(t *testing.T) {
+	testCases := []struct {
+		name          string
+		contextKey    string
+		contextValue  string
+		expectedToken string
+		expectedErr   string
+	}{
+		{
+			name:          "Good token",
```

Comment:
So this function is an easy one to test - great. I try to make the name of the test follow the test spec, like "it returns the expected token without error for a valid context value", but that's just a personal preference.

Reply:
Yup, totally right. I also like having meaningfulñ names for the tests, but I was too focused on writing them that I forgot to edit them before pushing. Thanks for the heads up!